quickannotator/api/v1/tile/helper.py

```diff
@@ -201,9 +123,9 @@ def remote_compute_on_tile(annotation_class_id: int, image_id: int, tile_id: int
         # Process the tile (using shapely for example)
         bbox = tilespace.get_bbox_for_tile(tile_id)
         bbox_polygon = Polygon([(bbox[0], bbox[1]), (bbox[2], bbox[1]), (bbox[2], bbox[3]), (bbox[0], bbox[3])])
-        for _ in range(random.randint(20, 40)):
-            polygon = generate_random_circle_within_bbox(bbox_polygon, 100)
-            insert_new_annotation(image_id, annotation_class_id, is_gt=False, tile_id=tile_id, polygon=polygon)
+        polygons = [generate_random_circle_within_bbox(bbox_polygon, 100) for _ in range(random.randint(20, 40))]
```

Comment:
i guess this is place holder code? : ) for sanity sake - can we mark it with "#TODO: [comment]" so that we can quickly grep for TODOs later on and make sure we don't miss anytihng?